crates/router/src/connector/adyen/transformers.rs

```diff
@@ -3957,8 +3957,10 @@ impl utils::MultipleCaptureSyncResponse for Response {
 pub struct AdyenPayoutCreateRequest {
     amount: Amount,
     recurring: RecurringContract,
+    selected_brand: Option<PayoutBrand>,
     merchant_account: Secret<String>,
-    bank: PayoutBankDetails,
+    bank: Option<PayoutBankDetails>,
+    additional_data: Option<PayoutAdditionalData>,
```

Comment:
Let's create enums for payment method data instead of making all fields optional